### ISSUE TITLE
The solution

### ISSUE THREAD

**Issue by MrWaltz-b14** [El-Qassam/Ooga-Booga-Python-Virus#4]
First, you dont get the CD so if the user does not execute or have their desktop area named desktop this entire script is well, useless.

**Comment by El-Qassam** [comment]
Ok bud. Get on the portal.

**Comment by MrWaltz-b14** [comment]
cd = os.getcwd() 
not a full solution lmao. my script is also looking better

**Comment by El-Qassam** [comment]
Agreed. Okay bud, this one’s just a start. I haven’t much time anyways to work on this. Citizenship celebration, Ramadan fasting fiesta, and becoming a bit more religious this year. Not much time, but I’ll try fixing it soon.
